Assignment2/printArray.js

```diff
@@ -0,0 +1,4 @@
+function printArray(array){
+  //show me the code!

```

Comment:
can be a simple arrow function: 

const printArray = (data) => (data.join(""));

Reply:
Actually we had to do task on the basis what we learnt through session. In session trainer didn't mentioned about arrow function so , that's why I haven't used it